styles.css

```diff
@@ -630,8 +638,26 @@ textarea::placeholder {
 }
 
 @media (max-width: 768px) {
-  .box3 {
-    display: none;
+  .nav-items {
+    width: 100%;
+    background-color: #6070ffcf;
+    position: fixed;
+    top: 0;
+    left: -100%;
+    height: 100%;
+    padding: 40px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .nav-items a {
+    color: white;
+    text-decoration: none;
+    font-size: 32px;
+  }
+
+  .close {
+    text-align: end;
   }
 
   #desktop {
```

Comment:
- [ ] Your mobile menu is working properly, but it is not covering the entire screen. It has a `margin-top` property at line 34 of this CSS file. Kindly fix this issue so the mobile menu can cover the entire screen like the Figma template.
 - [ ] I suggest reorganizing the code below this media query. Since this is a mobile first design, all code that should be applied to the mobile version should be placed before the media query that was opened to style the desktop version. This way you can have a more organized code.
 -Your design:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/61919617/188723619-a25236a4-7e65-45df-99a4-0dc2d111a5ab.png">
-Figma template:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/61919617/188723706-48628942-7791-4468-8133-a5df0bbb8e79.png">